switch to default rtd theme

anarcat opened the pull request:
still TODO:

* <del>add sidebar / find solution for "related links" ([support request upstream](https://github.com/snide/sphinx_rtd_theme/issues/254))</del> - moved to README for now
* <del>add back the "fork me on github" link?</del> just dropping the link for now
* <del>consider [other themes](https://github.com/bitprophet/alabaster)</del> alabaster doesn't support mobile devices
* <del>[add logo](https://github.com/snide/sphinx_rtd_theme/pull/69)</del> - needs a new logo, should probably be moved to a different issue

the results can be seen here for now:

http://borgbackup-anarcat.readthedocs.org/en/rtfd-debug/

anarcat commented:
another possible theme to consider: https://github.com/bitprophet/alabaster

for sidebars, maybe something here? http://sphinx-doc.org/config.html#confval-html_sidebars

it seems that we had custom sidebars that were shipped with our custom theme, not sure how to deal with that in the sphinx theme... maybe just move the sidebar to a new section? links.rst?

anarcat commented:
for the logo: https://github.com/snide/sphinx_rtd_theme/pull/69 but it would need to be something else than green...

anarcat commented:
sidebar issue: https://github.com/snide/sphinx_rtd_theme/issues/254

ThomasWaldmann commented:
alabaster seems to have some nice features, but is it responsive design to adapt to misc. devices / display sizes / dpi (that was one of the main reasons to move to rtd)?

"fork me on github" is not essential, we can also just drop that. developers will know anyway.

related links could be also put into README.rst, if sidebar extension isn't possible yet.

anarcat commented:
On 2015-10-10 06:37:08, TW wrote:
> alabaster seems to have some nice features, but is it responsive design to adapt to misc. devices / display sizes / dpi (that was one of the main reasons to move to rtd)?

i have no idea, would need to test / ask them. :)

> "fork me on github" is not essential, we can also just drop that. developers will know anyway.

agreed, i don't like that badge much anyways: "fork me" seems a little
ironic considering the history of borg... ;)

> related links could be also put into README.rst, if sidebar extension isn't possible yet.

okay then, maybe we can just start with that, it would be a good
addition to the README anyways.

a.
-- 
It will be a great day when our schools get all the money they need
and the air force has to hold a bake sale to buy a bomber.
                        - Unknown

anarcat commented:
alabaster seems to have some issues with mobile for now: https://github.com/bitprophet/alabaster/pull/57 and https://github.com/bitprophet/alabaster/issues/44 - i'd suggest sticking with rtd for now.

anarcat commented:
i have added the sidebar links to the README, i'd suggest moving the question of the logo to a separate issue for now.

otherwise this is ready to be merged, i updated my test copy with the new version:

http://borgbackup-anarcat.readthedocs.org/en/rtfd-debug/

ThomasWaldmann commented:
i did a quick test on my phone (6.4" FHD), looked good!

except the travis and coverage buttons, which get huge (full width) when the window width is below some specific value.

anarcat commented:
> except the travis and coverage buttons, which get huge (full width) when the window width is below some specific value.

is that a blocker? :)

ThomasWaldmann commented:
no

anarcat commented:
so i guess i'll just merge for now and create issues for the rest
